GitUI/UserControls/RevisionGrid/Columns/RevisionGraphColumnProvider.cs

```diff
@@ -505,7 +505,7 @@ string GetLaneInfo(int x, int rowIndex)
                     if (segmentsForLane.Count() == 1)
                     {
                         // Crossing lange
-                        laneInfoText.Append(segmentsForLane.First().Parent.GitRevision.Body ?? segmentsForLane.First().Parent.GitRevision.Subject);
+                        laneInfoText.Append(segmentsForLane.First().Parent.GitRevision?.Body ?? segmentsForLane.First().Parent.GitRevision?.Subject);
```

Comment:
Can `Parent` be `null`?

Reply:
No, a segment only exists with both a parent and a child.